storage/nfs-utils/pkg.yaml

```diff
@@ -0,0 +1,50 @@
+name: nfs-utils
+variant: scratch
+shell: /bin/bash
+dependencies:
+  - stage: base
+  - stage: libtirpc-zfs
+    from: /rootfs
+  - stage: libevent
+    from: /rootfs
+  - stage: sqlite
+    from: /rootfs
+steps:
+  - sources:
+      - url: https://mirrors.edge.kernel.org/pub/linux/utils/nfs-utils/{{ .NFS_UTILS_VERSION | replace "-" "." }}/nfs-utils-{{ .NFS_UTILS_VERSION | replace "-" "." }}.tar.xz
+        destination: nfs-utils.tar.xz
+        sha256: {{ .NFS_UTILS_SHA256 }}
+        sha512: {{ .NFS_UTILS_SHA512 }}
+    env:
+      PKG_CONFIG_PATH: /usr/local/lib/pkgconfig
+    prepare:
+      - |
+        tar -xf nfs-utils.tar.xz --strip-components=1
+
+        mkdir -p /usr/local/include/sys
+        cp /pkg/patches/sys-queue.h /usr/local/include/sys/queue.h
+
+        ./configure \
+          --disable-gss \
```

Comment:
since this is disabling everything else, is this only used as a binary in udev rule?

Reply:
are those needed?

Reply:
> are those needed?

No, not for what I need here. I removed everything else and only install the nfsrahead util inside the extension now.